src/pybind/mgr/progress/module.py

```diff
@@ -260,17 +259,49 @@ def __init__(self, *args, **kwargs):
 
     def _osd_out(self, old_map, old_dump, new_map, osd_id):
         affected_pgs = []
+        unmoved_pgs = []
         for pool in old_dump['pools']:
             pool_id = pool['pool']
             for ps in range(0, pool['pg_num']):
+                up_acting = old_map.pg_to_up_acting_osds(pool['pool'], ps)
+
+                # Was this OSD affected by the OSD going out?
+                old_osds = set(up_acting['up']) | set(up_acting['acting'])
+                was_on_out_osd = osd_id in old_osds
+                if not was_on_out_osd:
```

Comment:
no need to assign it to a variable, could just put
```py
if osd_id not in old_osds:
  continue
```

Reply:
done.